packages/terra-framework-docs/src/terra-dev-site/test/menu/menu/SelectableMenu.test.jsx

```diff
@@ -7,16 +7,21 @@ class SelectableMenu extends React.Component {
   constructor(props) {
     super(props);
     this.handleButtonClick = this.handleButtonClick.bind(this);
+    this.handleOnChange = this.handleOnChange.bind(this);
     this.handleRequestClose = this.handleRequestClose.bind(this);
     this.setButtonNode = this.setButtonNode.bind(this);
     this.getButtonNode = this.getButtonNode.bind(this);
-    this.state = { open: false };
+    this.state = { open: false, groupSelectedIndex: undefined };
```

Comment:
Nit: can we call this `selectedGroupIndex` since we're referring to the index of the selected item of an ItemGroup?

Reply:
Updated [here](https://github.com/cerner/terra-framework/pull/2046/commits/4d95a57681366c05247380fe7d0369f12d6960a9)

Reply:
Updated